.github/workflows/continuous-integration.yml

```diff
@@ -18,4 +21,5 @@ jobs:
         with:
           run: npm run test
       - name: Catch shell script issues
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
We can also run shell lint on macos afaik (and somehow maybe also on windows)

Reply:
hm not sure how, I'll check. I did run it on windows and mac and it failed both times, see above 3 commits)

Reply:
This is not necessary though. Since shellcheck is platform independent, I don't see a reason to run it three times.